RevenueCatUI/Templates/Template5View.swift

```diff
@@ -104,7 +109,6 @@ struct Template5View: TemplateViewType {
                 self.packages
             } else {
                 self.packages
-                    .padding(.top, self.defaultHorizontalPaddingLength)
```

Comment:
This + the changes of padding above make the padding consistent with template 2.